### PR TITLE
Jp add testing docs

### PR DESCRIPTION
**Pull request by jphilipps-eb**
Adding and updating our testing docs from main eng docs to this repo. Will add a link where old docs were to new docs location

### CODE REVIEW

**Comment by miglesiasEB** [comment]
LGTM

Thanks for this docs!

File: backbone/README.md [hunk 1]
```diff
@@ -368,3 +369,110 @@ return Marionette.ItemView.extend({
 ```
 
 **[⬆ back to top](#table-of-contents)**
+
+## Testable Modular JS with Backbone, Jasmine & Sinon
+
+### Reading
+
+-   Jasmine documentation <http://pivotal.github.com/jasmine/>
```
Comment (gagoar) [inline backbone/README.md]: we should point directly to the version we are using. 1.8.7 I think it is.

File: backbone/README.md [hunk 2]
```diff
@@ -368,3 +369,110 @@ return Marionette.ItemView.extend({
...
+        // we can assert that our spy was called
+        expect(userIsActivatedSpy).toHaveBeenCalledOnce();
+        this.server.restore();
+    });
+
+We have extended Sinon with a few custom functions, which you can see in django/media/django/js/tests/jasmine/spec/SpecHelper.js
+
+### Running the Jasmine Tests
+
+Go to <https://www.evbdev.com/js-tests/require> to run the entire JS unit test suite.
```
Comment (gagoar) [inline backbone/README.md]: the original path (where the index of every js tests exists is `https://www.evbdev.com/js-tests/` 
require/ is not the entire, is just what lives on `js/src/require` (and excluding `react` directory under that giving that react directoy have a different runner and a different jasmine version

File: backbone/README.md (hunk 1 above)
Comment (jphilipps-eb) [inline backbone/README.md]: good idea

File: backbone/README.md (hunk 2 above)
Comment (jphilipps-eb) [inline backbone/README.md]: ah right ok, will update

**Pull request merged**